pay-api/src/pay_api/resources/account.py

```diff
@@ -220,7 +220,7 @@ def post(account_number: str):
 
         # Check if user is authorized to perform this action
         required_roles = [Role.EDITOR.value, Role.VIEW_ALL_TRANSACTIONS.value] if view_all else [Role.EDITOR.value]
-        check_auth(business_identifier=None, account_id=account_number, all_of_roles=required_roles, is_premium=True)
+        check_auth(business_identifier=None, account_id=account_number, all_of_roles=required_roles, is_premium=False)
```

Comment:
you can just remove the `is_premium` arg here

Reply:
👍